### PR TITLE
core(cls): add learn more link

### PR DESCRIPTION
**Pull request by connorjclark**
@kaycebasques for determining if we should have a page for CLS specific to LH,  https://web.dev/lighthouse-cumulative-layout-shift/ ?

### CODE REVIEW

**Comment by kaycebasques** [comment]
Yes, we should. Linking to web.dev/cls is good enough for now. See https://github.com/GoogleChrome/web.dev/issues/2316 to track progress of the Lighthouse-specific guide.

**Comment by connorjclark** [comment]
Cool. I'll let this linger unmerged for now but if we must we'll merge this for 6.0 and update later.

**Comment by connorjclark** [comment]
Oh, we're hiding this by default so this is super puntable @kaycebasques

File: lighthouse-core/audits/metrics/cumulative-layout-shift.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ const UIStrings = {
   description: 'Cumulative Layout Shift is the sum of all layout shifts that occurred during a ' +
       'page\'s load. A layout shift is any movement an element makes once it is visible to the ' +
       'user. All layout shift is recorded, scored, and then aggregated into a cumulative score ' +
-      'between 0 and 1; 0 being a perfectly stable page, and >=0.5 being a highly shifting page.',
+      'between 0 and 1; 0 being a perfectly stable page, and >=0.5 being a highly shifting page.' +
```
Comment (paulirish) [inline lighthouse-core/audits/metrics/cumulative-layout-shift.js]:
```suggestion
      'between 0 and 1; 0 being a perfectly stable page, and >=0.5 being a highly shifting page. ' +
```

File: lighthouse-core/lib/i18n/locales/en-US.json [hunk 2]
```diff
@@ -861,7 +861,7 @@
     "message": "Manifest has a maskable icon"
   },
   "lighthouse-core/audits/metrics/cumulative-layout-shift.js | description": {
-    "message": "Cumulative Layout Shift is the sum of all layout shifts that occurred during a page's load. A layout shift is any movement an element makes once it is visible to the user. All layout shift is recorded, scored, and then aggregated into a cumulative score between 0 and 1; 0 being a perfectly stable page, and >=0.5 being a highly shifting page."
+    "message": "Cumulative Layout Shift is the sum of all layout shifts that occurred during a page's load. A layout shift is any movement an element makes once it is visible to the user. All layout shift is recorded, scored, and then aggregated into a cumulative score between 0 and 1; 0 being a perfectly stable page, and >=0.5 being a highly shifting page.[Learn more](https://web.dev/cls)."
```
Comment (paulirish) [inline lighthouse-core/lib/i18n/locales/en-US.json]:
```suggestion
    "message": "Cumulative Layout Shift is the sum of all layout shifts that occurred during a page's load. A layout shift is any movement an element makes once it is visible to the user. All layout shift is recorded, scored, and then aggregated into a cumulative score between 0 and 1; 0 being a perfectly stable page, and >=0.5 being a highly shifting page. [Learn more](https://web.dev/cls)."
```

File: lighthouse-core/lib/i18n/locales/en-XL.json [hunk 3]
```diff
@@ -861,7 +861,7 @@
...
+    "message": "Ĉúm̂úl̂át̂ív̂é L̂áŷóût́ Ŝh́îf́t̂ íŝ t́ĥé ŝúm̂ óf̂ ál̂ĺ l̂áŷóût́ ŝh́îf́t̂ś t̂h́ât́ ôćĉúr̂ŕêd́ d̂úr̂ín̂ǵ â ṕâǵê'ś l̂óâd́. Â ĺâýôút̂ śĥíf̂t́ îś âńŷ ḿôv́êḿêńt̂ án̂ él̂ém̂én̂t́ m̂ák̂éŝ ón̂ćê ít̂ íŝ v́îśîb́l̂é t̂ó t̂h́ê úŝér̂. Ál̂ĺ l̂áŷóût́ ŝh́îf́t̂ íŝ ŕêćôŕd̂éd̂, śĉór̂éd̂, án̂d́ t̂h́êń âǵĝŕêǵât́êd́ îńt̂ó â ćûḿûĺât́îv́ê śĉór̂é b̂ét̂ẃêén̂ 0 án̂d́ 1; 0 b̂éîńĝ á p̂ér̂f́êćt̂ĺŷ śt̂áb̂ĺê ṕâǵê, án̂d́ >=0.5 b̂éîńĝ á ĥíĝh́l̂ý ŝh́îf́t̂ín̂ǵ p̂áĝé.[L̂éâŕn̂ ḿôŕê](https://web.dev/cls)."
```
Comment (paulirish) [inline lighthouse-core/lib/i18n/locales/en-XL.json]:
```suggestion
    "message": "Ĉúm̂úl̂át̂ív̂é L̂áŷóût́ Ŝh́îf́t̂ íŝ t́ĥé ŝúm̂ óf̂ ál̂ĺ l̂áŷóût́ ŝh́îf́t̂ś t̂h́ât́ ôćĉúr̂ŕêd́ d̂úr̂ín̂ǵ â ṕâǵê'ś l̂óâd́. Â ĺâýôút̂ śĥíf̂t́ îś âńŷ ḿôv́êḿêńt̂ án̂ él̂ém̂én̂t́ m̂ák̂éŝ ón̂ćê ít̂ íŝ v́îśîb́l̂é t̂ó t̂h́ê úŝér̂. Ál̂ĺ l̂áŷóût́ ŝh́îf́t̂ íŝ ŕêćôŕd̂éd̂, śĉór̂éd̂, án̂d́ t̂h́êń âǵĝŕêǵât́êd́ îńt̂ó â ćûḿûĺât́îv́ê śĉór̂é b̂ét̂ẃêén̂ 0 án̂d́ 1; 0 b̂éîńĝ á p̂ér̂f́êćt̂ĺŷ śt̂áb̂ĺê ṕâǵê, án̂d́ >=0.5 b̂éîńĝ á ĥíĝh́l̂ý ŝh́îf́t̂ín̂ǵ p̂áĝé. [L̂éâŕn̂ ḿôŕê](https://web.dev/cls)."
```

File: lighthouse-core/test/results/sample_v2.json [hunk 4]
```diff
@@ -229,7 +229,7 @@
     "cumulative-layout-shift": {
       "id": "cumulative-layout-shift",
       "title": "Cumulative Layout Shift",
-      "description": "Cumulative Layout Shift is the sum of all layout shifts that occurred during a page's load. A layout shift is any movement an element makes once it is visible to the user. All layout shift is recorded, scored, and then aggregated into a cumulative score between 0 and 1; 0 being a perfectly stable page, and >=0.5 being a highly shifting page.",
+      "description": "Cumulative Layout Shift is the sum of all layout shifts that occurred during a page's load. A layout shift is any movement an element makes once it is visible to the user. All layout shift is recorded, scored, and then aggregated into a cumulative score between 0 and 1; 0 being a perfectly stable page, and >=0.5 being a highly shifting page.[Learn more](https://web.dev/cls).",
```
Comment (paulirish) [inline lighthouse-core/test/results/sample_v2.json]:
```suggestion
      "description": "Cumulative Layout Shift is the sum of all layout shifts that occurred during a page's load. A layout shift is any movement an element makes once it is visible to the user. All layout shift is recorded, scored, and then aggregated into a cumulative score between 0 and 1; 0 being a perfectly stable page, and >=0.5 being a highly shifting page. [Learn more](https://web.dev/cls).",
```

File: proto/sample_v2_round_trip.json [hunk 5]
```diff
@@ -434,7 +434,7 @@
             "title": "Avoid chaining critical requests"
         },
         "cumulative-layout-shift": {
-            "description": "Cumulative Layout Shift is the sum of all layout shifts that occurred during a page's load. A layout shift is any movement an element makes once it is visible to the user. All layout shift is recorded, scored, and then aggregated into a cumulative score between 0 and 1; 0 being a perfectly stable page, and >=0.5 being a highly shifting page.",
+            "description": "Cumulative Layout Shift is the sum of all layout shifts that occurred during a page's load. A layout shift is any movement an element makes once it is visible to the user. All layout shift is recorded, scored, and then aggregated into a cumulative score between 0 and 1; 0 being a perfectly stable page, and >=0.5 being a highly shifting page.[Learn more](https://web.dev/cls).",
```
Comment (paulirish) [inline proto/sample_v2_round_trip.json]:
```suggestion
            "description": "Cumulative Layout Shift is the sum of all layout shifts that occurred during a page's load. A layout shift is any movement an element makes once it is visible to the user. All layout shift is recorded, scored, and then aggregated into a cumulative score between 0 and 1; 0 being a perfectly stable page, and >=0.5 being a highly shifting page. [Learn more](https://web.dev/cls).",
```

**Pull request merged**